source/renderer/app/components/settings/categories/WalletsSettings.scss

```diff
@@ -79,3 +77,17 @@
     margin: 15px 0;
   }
 }
+
+:global
+  .WalletsSettings_component
+  .OptionsOverrides_options
+  :not(.SimpleOptions_hasSearch)
+  ul {
+  border-radius: 0 0 4px 4px;
+}
+
+:global {
+  .SimpleOptions_option em {
```

Comment:
@aleksandardjordjeviciohk this will affect ALL em elements within all of the `.SimpleOptions_option` we have in Daedalus. So, either move this declaration elsewhere (e.g. https://github.com/input-output-hk/daedalus/blob/develop/source/renderer/app/themes/overrides/OptionsOverrides.scss) or make it specific to the context of `.WalletSettings_component`

Reply:
Makes sense. I have updated it.

Reply:
I don't see the change @aleksandardjordjeviciohk?

Reply:
I would rather like to see you move this to the overrides scss...